### ISSUE TITLE
flow to storage error

### ISSUE THREAD

**Issue by vicanfon** [kelvins/mlflow-image#1]
running the docker-compose I got the minio and postgres running but I am getting the following error in the mlflow log:

(Background on this error at: https://sqlalche.me/e/14/e3q8)

Operation will be retried in 204.7 seconds

2021/10/15 10:53:19 WARNING mlflow.store.db.utils: SQLAlchemy engine could not be created. The following exception is caught.

" does not existionalError) FATAL:  database "mlflow

**Comment by kelvins** [comment]
Hello @vicanfon, thanks for reporting the issue.

Unfortunately, I was unable to replicate the issue on my local machine.

In fact, I'm getting some errors when starting Mlflow, for example:

```
mlflow      | (Background on this error at: https://sqlalche.me/e/14/e3q8)
mlflow      | Operation will be retried in 0.1 seconds
mlflow      | 2021/10/16 15:09:48 WARNING mlflow.store.db.utils: SQLAlchemy engine could not be created. The following exception is caught.
mlflow      | (psycopg2.OperationalError) could not connect to server: Connection refused
mlflow      | 	Is the server running on host "database" (172.18.0.3) and accepting
mlflow      | 	TCP/IP connections on port 5432?
```

But that's just a matter of waiting for the postgres container to be ready and then it is automatically resolved.

**Comment by kelvins** [comment]
I'm not sure, but by the error message you have shared, it looks like the "mlflow" database is not being created :thinking:

I'm wondering if this could be an issue related to file access control.

Could you please comment the volume mapping and try again? For example:

```
version: "3.7"
services:
  database:
    ...
    #volumes:
    #  - ./data:/var/lib/postgresql/data
    ...
```

**Comment by vicanfon** [comment]
Hi Kevin,

I commented those two and still the same error. I started thinking it was a communication problem between both containers but I can see that the database container is throwing the message:

" does not exist:28.166 UTC [116] FATAL:  database "mlflow

Pretty weird. Using a postgresql client I can see that the mlflow database is there. I don't know if the flow container is adding the quote attached to the name of the database... "flow. Pretty weird. I am running this on docker for windows.
